### PR TITLE
move qpidd reload to katello module

### PR DESCRIPTION
**Pull request by timogoebel**
This should go into the katello module. I haven't written the code for the module, yet. Currently, this is just a reminder.

### CODE REVIEW

**Comment by ekohl** [comment]
There are many cases like this where we need to move these notifies to other modules.

**Comment by timogoebel** [comment]
@ekohl : https://github.com/Katello/puppet-katello/pull/172 should move this to puppet-katello.

File: manifests/candlepin.pp [hunk 1]
```diff
@@ -80,7 +80,6 @@
       client_cert => $client_cert,
       refreshonly => true,
       subscribe   => Exec['create-nss-db'],
-      notify      => Service['qpidd'],
```
Comment (ehelms) [inline manifests/candlepin.pp]: I am a little surprised this was needed at all. I haven't tested to verify, but it seems like qpidd shouldn't have to be restarted every time the nssdb is updated.

**Comment by ehelms** [comment]
@ekohl The katello PR was merged, what else do you think this needs?

**Comment by ekohl** [comment]
There is another qpid refresh a bit lower (line 93) that you will run into as well.

**Comment by ekohl** [comment]
@timogoebel could you look at the other refresh as well? After that I think this can be merged.

**Comment by timogoebel** [comment]
@ekohl : I was hoping to get rid of the other qpid refresh by https://github.com/Katello/puppet-certs/pull/134
I think, this one has to go first as the exec actually needs the service running.

**Pull request merged**